HIP/hip-0000-mirror-node-account-id.md

```diff
@@ -0,0 +1,68 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Message Account Id in Mirror Node API Response
+- author: scalemaildev
+- type: Standards Track
+- category: Mirror
+- status: Draft
+- created: 2021-10-18
+- discussions-to: <a URL pointing to the official discussion thread>
```

Comment:
Please create a new discussion thread in the `Discussions` tab and link to it here